
{'date': 'Tue Aug 06 2019 19:20:19.25 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 18
  class doidera()
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Aug 06 2019 19:20:38.960 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    doidera()
  module <module> line 22
    dr_zukman-Elemento(img=DR_ZUKMAN)
NameError: name 'dr_zukman' is not defined
'''},
{'date': 'Tue Aug 06 2019 19:20:55.27 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    doidera()
  module <module> line 26
    dr_zukmam.entra(congresso.norte)
NameError: name 'dr_zukmam' is not defined
'''},
{'date': 'Tue Oct 01 2019 18:48:29.231 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 2
  from _spy.vitollino.main import Cena, Texto,  """Elemento """, STYLE,Sala
                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 01 2019 18:52:51.227 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  class Personagens():
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Oct 01 2019 18:53:57.903 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  def entra_porta(self,evento,nome):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Oct 01 2019 18:58:15.74 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Cenas()
  module <module> line 43
    ana_maria.entra(congresso.norte)
NameError: name 'ana_maria' is not defined
'''},
{'date': 'Tue Oct 01 2019 19:00:15.761 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Cenas()
  module <module> line 46
    dr_zukman.entra(congresso.norte)
NameError: name 'dr_zukman' is not defined
'''},
{'date': 'Tue Oct 01 2019 19:00:53.616 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Cenas()
  module <module> line 47
    porta.entra(congresso.norte)
AttributeError: 'porta' object has no attribute 'entra'
'''},
{'date': 'Tue Oct 01 2019 19:03:51.0 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 54
    Cenas()
  module <module> line 49
    Porta().entra(congresso.norte)
  module <module> line 38
    dragger=dict(Aninha=self.entra_porta)
AttributeError: 'Porta' object has no attribute 'entra_porta'
'''},
{'date': 'Wed Oct 30 2019 08:00:41.998 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 59
  oi
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Jan 06 2020 19:23:57.251 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.ana_maria= Elemento(img=imagem, tit=nome, drag=True, style=dict(
                                                                                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jan 06 2020 19:29:03.330 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.ana_maria= Elemento(img=imagem, tit=nome, drag=True, style=dict(
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jan 06 2020 19:40:22.55 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 30
  left=400,top=200,width,350,heigth="400px"))
                        ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Mon Jan 06 2020 20:58:19.271 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  LETICIA = "https://i.imgur.com/Oo1sn9s.png
                                            ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Jan 06 2020 21:50:06.740 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  LETICIA = "https://i.imgur.com/Oo1sn9s.png
                                            ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Jan 06 2020 21:50:18.959 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 77
  self.c=Elemento(img= "https://i.imgur.com/MUrxyGb.png", tit="Oiiii, meu nome é Gabi. 
                                                                                       ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Jan 06 2020 22:42:24.694 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  self.a.vai():
               ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jan 07 2020 00:35:58.94 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 101
  def vai(self,ev=0):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Jan 07 2020 00:36:11.656 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 111
    mendel()            
  module <module> line 95
    self.a.entra(congresso.oeste)
NameError: name 'congresso' is not defined
'''},
{'date': 'Tue Jan 07 2020 00:36:19.764 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 111
    mendel()            
  module <module> line 95
    self.a.entra(congresso.oeste)
NameError: name 'congresso' is not defined
'''},
{'date': 'Tue Jan 07 2020 00:56:32.661 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 169
    doidera()    
  module <module> line 33
    c=mendel(congress0)
NameError: name 'congress0' is not defined
'''},
{'date': 'Tue Jan 07 2020 01:27:11.564 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 88
  self.gabi.vai=self.congresso sul
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jan 07 2020 15:52:31.954 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 79
    Cenas()
TypeError: __init__() missing 1 positional argument: congresso
'''},
{'date': 'Tue Jan 07 2020 15:53:00.671 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 172
    TakeFioCruz()
  module <module> line 34
    c=Mendel(congresso)
  module <module> line 84
    self.foi,congresso.oeste.vai=congresso.SALA3.vai,self.vai
AttributeError: 'Sala' object has no attribute 'SALA3'
'''},
{'date': 'Tue Jan 07 2020 15:54:33.73 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 172
    TakeFioCruz()
  module <module> line 36
    a=Cenas(congresso)
  module <module> line 72
    PERSONAGEM.entra(congresso.norte)
AttributeError: 'NoneType' object has no attribute 'entra'
'''},
{'date': 'Tue Jan 07 2020 15:57:28.660 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 173
    TakeFioCruz()
  module <module> line 33
    a=Cenas(congresso)
UnboundLocalError: local variable 'congresso' referenced before assignment
'''},
{'date': 'Tue Jan 07 2020 15:58:05.377 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 173
    TakeFioCruz()
  module <module> line 34
    a=Cenas(congresso)
  module <module> line 73
    PERSONAGEM.entra(congresso.norte)
AttributeError: 'NoneType' object has no attribute 'entra'
'''},
{'date': 'Tue Jan 07 2020 16:01:03.996 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 173
    TakeFioCruz()
  module <module> line 35
    a=Cenas(congresso, d)
  module <module> line 71
    self,personagem = personagem
TypeError: 'Personagens' object is not iterable
'''},
{'date': 'Tue Jan 07 2020 16:04:32.301 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 174
    TakeFioCruz()
  module <module> line 35
    a=Cenas(congresso, d)
  module <module> line 76
    Porta(self,personagem).entra(congresso.norte)
TypeError: __init__() takes 2 positional arguments but more were given
'''},
{'date': 'Tue Jan 07 2020 16:08:22.953 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 174
    TakeFioCruz()
  module <module> line 43
    e=Morboletas(congresso)
NameError: name 'Morboletas' is not defined
'''},
{'date': 'Tue Jan 07 2020 16:11:48.414 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 174
    TakeFioCruz()
  module <module> line 36
    congresso.norte.vai=a.vai
AttributeError: 'Cenas' object has no attribute 'vai'
'''},
{'date': 'Tue Jan 07 2020 16:15:46.263 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 178
    TakeFioCruz()
  module <module> line 39
    congresso.norte.vai=d.vai
AttributeError: 'Personagens' object has no attribute 'vai'
'''},
{'date': 'Tue Jan 07 2020 16:28:10.540 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 183
    TakeFioCruz()
  module <module> line 35
    a=Cenas(congresso, d)
  module <module> line 79
    self.vai()
  module <module> line 88
    Texto(self.congresso.norte, "Bem vindos à FioCruz").vai()
AttributeError: 'Cenas' object has no attribute 'congresso'
'''},